andela/travel_tool_front
# #162376432 Add component for displaying error messages

#### What does this PR do?
- Add a page not found the component 

#### Description of Task to be completed?
- Add component for displaying an error message when a user tries to access wrong routes

#### How should this be manually tested?
- Setting up the backend
     - Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
     - Enter into the project directory - `cd travel_tool_back`
     - checkout the branch git checkout `develop`
     - Install dependencies using `yarn install`
     - Rollback and run database migrations with `yarn db:rollmigrate`
     - start the backend server with `yarn run start:dev`


- Setting up the frontend
   - Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
   - Enter into the project directory - `cd travel_tool_front`
   - Checkout to the branch `ft-404-error-handlers-162376432` then run the app `yarn start`
   - Login to the application
   - Navigate to the `/fake_url` or any unknown route on the application
   - It is expected that the page not found component will be rendered.

#### Any background context you want to provide?
- All routes that are not configured in the `Routes` will render this new component

#### What are the relevant pivotal tracker stories?
[#162376432](https://www.pivotaltracker.com/story/show/162376432)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-12-06 at 23 53 40" src="https://user-images.githubusercontent.com/21138053/49611312-35fbf600-f9b2-11e8-969a-a9208efdaea1.png">
